packages/stream_chat/lib/src/api/websocket.dart

```diff
@@ -107,43 +107,43 @@ class WebSocket {
       _connectionStatusController.add(status);
 
   /// The current connection status value
-  ConnectionStatus get connectionStatus => _connectionStatusController.value;
+  ConnectionStatus? get connectionStatus => _connectionStatusController.value;
 
   /// This notifies of connection status changes
   Stream<ConnectionStatus> get connectionStatusStream =>
       _connectionStatusController.stream;
 
-  String _path;
+  late final String _path;
```

Comment:
I'm observing:

```
LateInitializationError: Field '_path@2300330865' has already been initialized.
```

because `_path` is updated multiple times elsewhere in this class.

Reply:
Thanks! That was not right, I just updated it